core/sql/executor/hiveHook.cpp

```diff
@@ -319,7 +319,7 @@ struct hive_sd_desc* populateSD(HiveMetaData *md, Int32 mainSdID,
   
   // TODO : loop over SDs
   if (findAToken(md, tblStr, pos, "sd:StorageDescriptor(", 
-                 "getTableDesc::sd:StorageDescriptor(###"))
+                 "getTableDesc::sd:StorageDescriptor(###"),FALSE)
```

Comment:
Ok looks like the right fix then. We don't want this error text to show up for some other NATable error.